wikidata/client.py

```diff
@@ -177,21 +178,30 @@ def decode_datavalue(self,
                       self.datavalue_decoder)
         return decode(self, datatype, datavalue)
 
-    def request(self, path: str) -> Union[
+    def request(self, path: str) -> Optional[Union[
         bool, int, float, str,
         Mapping[
             str,
             Union[bool, int, float, str, Mapping[str, object], Sequence]
         ],
         Sequence[Union[bool, int, float, str, Mapping[str, object], Sequence]]
-    ]:
+    ]]:
```

Comment:
[According to the `typing` API docs][1]:

> `Optional[X]` is equivalent to `Union[X, None]`.

Therefore, you don't have to put `Union[A, B, ...]` inside `Optional[...]`.  Instead, we could write the equivalent code like `Union[A, B, ..., None]`.

[1]: https://docs.python.org/3/library/typing.html#typing.Optional